frontend/.env

```diff
@@ -0,0 +1,3 @@
+VUE_APP_TITLE=Business Partner Agent
+VUE_APP_API_BASE_URL=/api
+VUE_APP_EVENTS_PATH=events
```

Comment:
Does this really work for the production build served by the BPA (Micronaut) backend?
I can only imagine this to work if you launch a second frontend with `npm run serve`. 
I'll try to check it myself later.

Reply:
Yes, it works when compiled for production.
In muy local development, I always run one BPA in production mode.

Reply:
Cool! Didn't know that.

Reply:
ok, but this replacement happens by npm during the frontend build process because it evaluates the .env file. this is not something that happens during runtime. so we are mixing runtime and build time config here. this might get confusing for the user, because the rest of the config implies that you simply change a value and restart for the changes to take effect. here one needs to rebuild, or am i getting this wrong?